exercises/diffie-hellman/src/lib.rs

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
an extra newline that I believe could be removed here

Reply:
(ignore the fact that this comment shows in the wrong place. The newline I intended has been removed, and the one this comment is currently on should not be removed)

Reply:
Yes, then it may be better if you could do the branch/commit cleaning.